unicoremc/views.py

```diff
@@ -23,7 +24,9 @@
 from unicoremc import tasks, utils
 
 
-def get_all_repos(request):
+def get_repos(refresh):
+    if not refresh:
+        return cache.get('repos')
```

Comment:
Maybe it should check if it is in redis? Otherwise this method will return None.
```
val = cache.get('repos')
if val is not None:
     return val
```